Bottom: ef12221b9b980010ecf33567429f0b17dee2a194
Top:    234253d4468a10b90d3f39628af673d6b3b937fb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 13:07:16 +0100

Refresh of temporary-debug-for-toporouter

---

diff --git a/src/const.h b/src/const.h
index c82e90a..f554fd4 100644
--- a/src/const.h
+++ b/src/const.h
@@ -68,10 +68,10 @@
 #define LN_2_OVER_2		0.346573590
 
 /* PCB/physical unit conversions */
-#define COORD_TO_MIL(n)	((n) / 25400.0)
-#define MIL_TO_COORD(n)	((n) * 25400.0)
-#define COORD_TO_MM(n)	((n) / 1000000.0)
-#define MM_TO_COORD(n)	((n) * 1000000.0)
+#define COORD_TO_MIL(n)	((n) / 100.0)
+#define MIL_TO_COORD(n)	((n) * 100.0)
+#define COORD_TO_MM(n)	((n) * 0.000254)
+#define MM_TO_COORD(n)	((n) / 0.000254)
 #define COORD_TO_INCH(n)	(COORD_TO_MIL(n) / 1000.0)
 #define INCH_TO_COORD(n)	(MIL_TO_COORD(n) * 1000.0)
